### DISCUSSION TITLE
Open invite to join Geeksblabla episode about the challenge

### DISCUSSION THREAD

**Discussion by aboullaite** [geeksblabla/blanat#303] [Announcements]
Hello everyone, 
As we are trying to pull out the final numbers and announce the winner, we want to express our gratitude to all you contribution. The participation exceeded our expectations and we were enjoyably overwhelmed with your submissions. We are all winners 🥳  we all learned something along the way and that was the main goal of this initiative.

We are planning to hav an episode very soon about the challenge and how it went and defirent optimization techniques you followed and we'd love to have you onboard. We will try to have as much as we can if possible, but we will limit it to 7/9 participants. Looking forward hearing from you

**Comment by k34n4y138** [comment]
Has the time of the episode been decided yet? and will the Leaderboard be updated before or through it?

**Comment by hamzamogni** [comment]
I would like to thank the whole Geeksblabla team for this challenge, especially @aboullaite @chermehdi and @adnanmerrakchi . This was a fun and enjoyable challenge that kept me awake for days. Learned a lot along the way. Kudos to everyone involved.

Best Community ♥️

**Comment by aboullaite** [comment]
Calling some names here if you want to join us this sunday: @samir- @webNeat @yousfiSaad @j-mounim @ismail-bertalfilali @essmehdi

**Reply by essmehdi** [reply]
Yes, I would like to join.

**Reply by aboullaite** [reply]
Can you please share with us your email to send you an invite in one of @geeksblabla channels ? X (preferably)/ insta / facebook

**Reply by essmehdi** [reply]
Done! Did it via Insta. DMs in X are behind a  paywall 😬.

**Reply by adnanmerrakchi** [reply]
@essmehdi @samir- @webNeat @yousfiSaad @j-mounim @ismail-bertalfilali @ismail-bertalfilali  Please, could you send your picture to include it in the episode poster? by email adnanmerrakchi@gmail.com or text me on twitter https://twitter.com/adnanmrakchi ?

**Reply by yousfiSaad** [reply]
Here is my email: yousfi.saad@gmail.com

**Reply by adnanmerrakchi** [reply]
i just sent you an email.
For people who already sent it to me on twitter, just ignore it. thank you

**Reply by j-mounim** [reply]
salam alaykom
@aboullaite mou7al ne9dar ne7dar adna wahed lmonassaba walakin radi n7awel ndir chi salta la 9darte
ta7iya lil jami3 w chokran 3ala lmajhoudate w 3la wa9tkom.
l3gouba lta7adi jay inchallah.

**Reply by aboullaite** [reply]
Ila 9darti tsayfat l @adnanmerrakchi email dyalik so he can send you an invite. Merhba bik if you can

**Comment by ismail-bertalfilali** [comment]
Hello guys,

Great work, everyone, and exceptional organization as well. Congratulations to @k34n4y138 for the well-deserved first place.

Thank you for the invite. Unfortunately, I cannot be there with you, but I will certainly watch the replay of the video.

**Comment by aboullaite** [comment]
Thank you all for joining us and the fun
